src/main/java/duke/models/LockerList.java

```diff
@@ -0,0 +1,52 @@
+package duke.models;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LockerList {
+
+    private List<Locker> lockerList;
+
+    public LockerList(List<Locker> lockerList) {
+        this.lockerList = lockerList;
+    }
+
+    public LockerList() {
+        lockerList = new ArrayList<>();
+    }
+
+    public void addLocker(Locker locker) {
+        lockerList.add(locker);
+    }
+
+    public void deleteLocker(Locker locker) {
+        lockerList.remove(locker);
+    }
+
+    public Locker getLocker(int index) {
+        return lockerList.get(index);
+    }
+
+    public int numLockers() {
+        return lockerList.size();
+    }
+
+    public List<Locker> getAllLockers() {
+        return lockerList;
+    }
+
+
+    @JsonGetter("lockers")
+    public List<Locker> getLockerList() {
+        return lockerList;
```

Comment:
Returning List will pass by reference causing the object that calls 

`getLockerList()` to tamper with the `lockerList`, may want to reconsider to create a copy of `lockerList`